ggml-cuda/convert.cu

```diff
@@ -707,7 +707,7 @@ static void dequantize_row_iq4_xs_cuda(const void * vx, dst_t * y, const int64_t
 
 template <typename src_t, typename dst_t>
 static __global__ void convert_unary(const void * __restrict__ vx, dst_t * __restrict__ y, const int64_t k) {
-    const int64_t i = (int64_t)blockDim.x*blockIdx.x + threadIdx.x;
+    const int i = blockDim.x*blockIdx.x + threadIdx.x;
```

Comment:
Same question.

Reply:
These were all originally `int` and I reverted them to avoid changing more than needed.

Reply:
No they were not. Go to the "files changed" tab and look at the combined changes of all of your commits relative to master.

Reply:
I changed them in PR https://github.com/ggerganov/llama.cpp/pull/6491.